apps/discord-bot/package.json

```diff
@@ -48,6 +50,8 @@
     "dotenv": "^17.2.0",
     "express": "5.1.0",
     "filesize": "10.1.6",
+    "fs": "0.0.1-security",
+    "fsp": "^0.1.2",
```

Comment:
* `fs` is a built in Node module and does not need to be declared as a dependency (in fact, the version here indicates this package is really a placeholder to prevent bad actors from creating a fake npm package spoofing the name.)
* fsp provides a Promise based API to the fs module, but I don't see it being used anywhere. I've suggested using Promises in other comments, but Node now has a built in Promise API with the `fs/promises` module.